R/stats_methods.R

```diff
@@ -67,24 +67,25 @@ confint.fh <- function(object, parm = NULL, level = 0.95, ...) {
   }
 }
 
-# Extract the AIC from a model fit of an emdi object ---------------------------
+# Extract the AIC from a Model Fit of an emdi Object ---------------------------
 #' @export
 #' @method extractAIC fh
 #' @importFrom stats extractAIC
 
 extractAIC.fh <- function(fit, ...) {
   throw_class_error(fit, "fh")
   if (!is.null(fit$model$model_select$AIC)) {
-    message(paste0(
-      "Estimation approach used is ", fit$method$method, ": ",
-      round(fit$model$model_select$AIC, 5)
-    ))
+    message(strwrap(prefix = " ", initial = "",
+                    paste0("Estimation approach used is ",
+                           fit$method$method, ": ",
+                           round(fit$model$model_select$AIC, 5)
+                           )))
```

Comment:
```suggestion
    message(strwrap(prefix = " ", initial = "",
                    sprintf("Estimation approach used is %s: %0.5g",
                           fit$method$method, 
                           round(fit$model$model_select$AIC, 5)
                           )))
```
with `0.5g`, up to 5 significant figures are printed, and exponential syntax is used for very small / large numbers. Is this beneficial?